container/docker/fsHandler.go

```diff
@@ -66,12 +69,12 @@ func (fh *realFsHandler) needsUpdate() bool {
 
 func (fh *realFsHandler) update() error {
 	// TODO(vishh): Add support for external mounts.
-	baseUsage, err := fh.fsInfo.GetDirUsage(fh.rootfs)
+	baseUsage, err := fh.fsInfo.GetDirUsage(fh.rootfs, duTimeout)
 	if err != nil {
 		return err
 	}
 
-	extraDirUsage, err := fh.fsInfo.GetDirUsage(fh.extraDir)
+	extraDirUsage, err := fh.fsInfo.GetDirUsage(fh.extraDir, duTimeout)
```

Comment:
I think we should log these errors at `Error` level (here: https://github.com/vishh/cadvisor/blob/timeout-du/container/docker/fsHandler.go#L99 - I can't figure out how to comment there on GH)